CMakeLists.txt

```
@@ -35,6 +35,7 @@ option(HIGHFIVE_USE_OPENCV "Enable OpenCV testing" ${USE_OPENCV})
 option(HIGHFIVE_UNIT_TESTS "Enable unit tests" ON)
 option(HIGHFIVE_EXAMPLES "Compile examples" ON)
 option(HIGHFIVE_PARALLEL_HDF5 "Enable Parallel HDF5 support" OFF)
+option(HIGHFIVE_DOC "Enable documentation building" ON)
```

```suggestion
option(HIGHFIVE_BUILD_DOCS "Enable documentation building" ON)
```
…would be more fitting, IMO.